modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ViewSpaceSitesAbstractSectionDisplayContext.java

```diff
@@ -72,12 +75,48 @@ public Map<String, Object> getEmptyState() {
 		).build();
 	}
 
+	public List<FDSActionDropdownItem> getFDSActionDropdownItems() {
+		return ListUtil.fromArray(
+			new FDSActionDropdownItem(
+				StringBundler.concat(
+					"/o/headless-asset-library/v1.0/asset-libraries/", _groupId,
+					"/sites/{id}"),
+				null, "delete",
+				_language.get(_httpServletRequest, "disconnect"), "delete",
+				null, "headless"),
+			_getSearchableFDSActionDropdownItem(true),
+			_getSearchableFDSActionDropdownItem(false));
+	}
+
 	public Map<String, Object> getHeaderProps() throws Exception {
 		return SpaceAbstractHeaderUtil.getSpaceAbstractHeaderProps(
 			_httpServletRequest, "view-all-sites", _getSpaceSitesHeaderTitle(),
 			StringPool.BLANK);
 	}
 
+	private FDSActionDropdownItem _getSearchableFDSActionDropdownItem(
+		boolean searchable) {
+
+		FDSActionDropdownItem fdsActionDropdownItem = new FDSActionDropdownItem(
+			StringBundler.concat(
+				"/o/headless-asset-library/v1.0/asset-libraries/", _groupId,
+				"/sites/{id}"),
+			null, searchable ? "make-searchable" : "make-unsearchable",
+			_language.get(
+				_httpServletRequest,
+				searchable ? "make-searchable" : "make-unsearchable"),
+			"put", null, "headless");
+
+		if (searchable) {
+			fdsActionDropdownItem.setRequestBody("{\"searchable\": true}");
+		}
+		else {
+			fdsActionDropdownItem.setRequestBody("{\"searchable\": false}");
+		}
```

Comment:
Can you change this to `fdsActionDropdownItem.setRequestBody("...." + Boolean.toString(searchable) + "}");`?